bin/include/jvmdefaults.bat

```diff
@@ -89,6 +89,7 @@ if %java_version% GEQ 15 (
     --add-opens=java.base/java.lang.invoke=ALL-UNNAMED ^
     --add-opens=java.base/java.math=ALL-UNNAMED ^
     --add-opens=java.sql/java.sql=ALL-UNNAMED ^
+    --add-opens=java.sql/java.time=ALL-UNNAMED ^
```

Comment:
There is no such module: `java.sql/java.time`

https://github.com/openjdk/jdk/blob/jdk-17%2B35/src/java.sql/share/classes/module-info.java